Packs/CTFCloud/Scripts/APIKeyRevokeCTF/APIKeyRevokeCTF.yml

```diff
@@ -0,0 +1,20 @@
+args:
+- name: accesskey
+  required: true
+comment: CTF script that initiates a revoke for all user access keys.
```

Comment:
```suggestion
comment: CTF script that initiates a revoke an access key.
```

Reply:
I suggest:
CTF script that initiates revoking an access key.